operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -258,117 +328,64 @@ HPE Cray EX System Admin Toolkit (SAT) product stream documentation (`S-8031`) f
+    ```text
+    WARNING: The cray-sdu-rda container is not running on ncn-m001.
+    WARNING: The cray-sdu-rda container is not running on ncn-m002.
+    WARNING: The cray-sdu-rda container is not running on ncn-m003.
+    ```
 
-        ```bash
-        kubectl -n services describe job <BOA Job ID> |grep ConfigMap -A 1 -B 1
-        ```
+1. (`ncn-mw#`) Cancel the running BOS sessions.
 
-        Example:
+    1. Identify the BOS Sessions to delete.
 
         ```bash
-        kubectl -n services describe job boa-0216d2d9-b2bc-41b0-960d-165d2af7a742 |grep ConfigMap -A 1 -B 1
-           boot-session:
-            Type:      ConfigMap (a volume populated by a ConfigMap)
-            Name:      e0543eb5-3445-4ee0-93ec-c53e3d1832ce    <<< ConfigMap name. Delete this one.
-        --
-           ca-pubkey:
-            Type:      ConfigMap (a volume populated by a ConfigMap)
-            Name:      cray-configmap-ca-public-key
+        cray bos sessions list --format json
         ```
 
-        Delete the ConfigMap associated with the `boot-session`, not the `ca-pubkey`.
-
-        To delete the ConfigMap:
+    1. Delete each running BOS session.
 
         ```bash
-        kubectl -n services delete cm <ConfigMap name>
+        cray bos sessions delete <session ID>
         ```
 
         Example:
 
         ```bash
-        kubectl -n services delete cm e0543eb5-3445-4ee0-93ec-c53e3d1832ce
-        configmap "e0543eb5-3445-4ee0-93ec-c53e3d1832ce" deleted
-        ```
-
-    1. Delete the BOA jobs.
-
-        ```bash
-        kubectl -n services delete job <boa-job-id>
-        ```
-
-        This will kill the BOA job and the BOS session associated with it.
-
-        When a job is killed, BOA will no longer attempt to execute the operation it was attempting to perform. This does not mean that
-        nothing continues to happen. If BOA has instructed a node to power on, the node will continue to power even after the BOA job
-        has been killed.
-
-    1. Delete the BOS session.
-        BOS keeps track of sessions in its database. These entries need to be deleted.
-        The BOS Session ID is the same as the BOA Job ID minus the prepended `boa-`
-        string. Use the following command to delete the BOS database entry.
-
-        ```bash
-        cray bos v1 session delete <session ID>
+        cray bos sessions delete 0216d2d9-b2bc-41b0-960d-165d2af7a742
         ```
 
-        Example:
+1. Coordinate with the site system administators to prevent new sessions from starting in the services listed.
 
-        ```bash
-        cray bos v1 session delete 0216d2d9-b2bc-41b0-960d-165d2af7a742
-        ```
+    There is no method to prevent new sessions from being created as long as the service APIs are accessible on the API gateway.
 
-1. Coordinate with the site to prevent new sessions from starting in the services listed.
+1. Follow the vendor workload manager documentation to drain processes running on compute nodes.
```

Comment:
Should this step (or its substeps) have an indicator of which node it should be run on?

Reply:
That depends on how the WLM is installed.  Some customers do not use HPE Slurm or HPE PBS Pro running in a container on a worker node.  They may use bare-metal on a worker node or an application node or completely external to the system.  We cannot know where they should run the command which controls the WLM scheduler.